CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ set(RECASTNAVIGATION_EXAMPLES OFF CACHE BOOL "" FORCE)
 # Disabled no-register
 # Disabled unknown pragmas because Linux doesn't understand Windows pragmas.
 if(UNIX)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++17 -O2 -Wuninitialized -fPIC")
```

Comment:
just curious as to why this was changed, I thought that we were already on c++20

Reply:
It's an extraneous flag.  We already set the version above this with cxx standard.